molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/.yamllint

```diff
@@ -1,3 +1,5 @@
+---
+# Based on ansible-lint config
```

Comment:
Any ideas how to keep it from going out of date again?

Reply:
@decentral1se  No, not really. Unless part of the build process we grab whatever galaxy uses. But I think the new one basically helps catch errors that also appear in Ansible-Lint as some of the YAML checks are the same we'd get in Ansible-Lint